### PR TITLE
Add gnuradio

### LINKED ISSUE #10615: Package request: GNU Radio

### PR DESCRIPTION
**Pull request by abenson**
Closes #10615

### CODE REVIEW

File: srcpkgs/volk/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+hostmakedepends="pkg-config"
+makedepends="python-devel python-cheetah boost-devel"
+license="GPL-3"
+homepage="http://libvolk.org/"
+distfiles="https://github.com/gnuradio/volk/archive/v${version}.tar.gz"
+checksum=90b69515d3cc6d76bbc589cec63d600b690bffe2e2938be2f895176f18f7e3af
+
+pre_configure() {
+	export CFLAGS="-D_GLIBCXX_USE_CXX11_ABI=0"
```
Comment (pullmoll) [inline srcpkgs/volk/template]: You should define `CFLAGS="-D_GLIBCXX_USE_CXX11_ABI=0"` globally instead, i.e. at the template scope.

File: common/shlibs [hunk 2]
```diff
@@ -3091,3 +3091,9 @@ libscrypt.so.0 libscrypt-devel-1.21_1
 liblwipv6.so.2 lwipv6-1.5a_1
 libpipewire-0.1.so.0 pipewire-0.1.7_1
 libspa-lib.so.0 pipewire-0.1.7_1
+libvolk.so.1.3 volk-1.3_1
```
Comment (pullmoll) [inline common/shlibs]: Perhaps you can move this addition into the commit which creates the volk package?

File: srcpkgs/volk/template (hunk 1 above)
Comment (abenson) [reply srcpkgs/volk/template]: This is outdated, I patched the CMakeLists.txt, but didn't remove it from pre_configure.

File: common/shlibs (hunk 2 above)
Comment (abenson) [reply common/shlibs]: Whoops, wrong commit. Thanks for catching.

**Comment by abenson** [comment]
It should all be correct now.

**Comment by pullmoll** [comment]
Fine, thank you!
vy 73 DG9KBM

**Pull request closed without merging**